aspnetcore/signalr/authn-and-authz.md

```diff
@@ -74,6 +72,20 @@ public void Configure(IApplicationBuilder app)
 
 ::: moniker-end
 
+::: moniker range="< aspnetcore-6.0"
+
+> [!NOTE]
+> If a token expires during the lifetime of a connection, the connection will continue to work. LongPolling and ServerSentEvents connections will fail on subsequent requests if they don't send newer access tokens.
```

Comment:
```suggestion
> If a token expires during the lifetime of a connection, the connection continues to work. `LongPolling` and `ServerSentEvent` connections fail on subsequent requests if they don't send new access tokens.
```

Reply:
Avoid FT Future Tense

Reply:
`LongPolling` and `ServerSentEvent`  are probably wrong but they're not English words that will MT (machine translate)